playbooks/roles/mattermost/templates/config.json.j2

```diff
@@ -128,17 +126,16 @@
         "EnableSignInWithUsername": true,
         "SendEmailNotifications": true,
         "RequireEmailVerification": false,
-        "FeedbackName": "Mattermost Notification",
+        "FeedbackName": "ND MM Notification",
```

Comment:
I might be misunderstanding something, but we use the mattermost role for chat.opencraft.com too